test/suites/client-LocalSettings.php

```diff
@@ -1,39 +1,13 @@
 <?php
 
-ini_set( 'display_errors', 1 );
-$wgShowExceptionDetails = true;
-$wgShowSQLErrors = true;
-$wgDebugDumpSql  = false;
-$wgShowDBErrorBacktrace = false;
-$wgEnableParserCache = false;
-$wgCachePages = false;
-$wgParserCacheType = CACHE_NONE;
-$wgMainCacheType = CACHE_NONE;
-
-## Logs
-## Save these logs inside the container
-$wgDebugLogFile = '/var/log/mediawiki/mw.client.debug.log';
-$wgDebugLogGroups = array(
-	'resourceloader' => '/var/log/mediawiki/mw.client.resourceloader.log',
-	'exception' => '/var/log/mediawiki/mw.client.exception.log',
-	'error' => '/var/log/mediawiki/mw.client.error.log',
-	'fatal' => '/var/log/mediawiki/mw.client.fatal.log',
-);
-
-## Site Settings
 $wgMetaNamespace = "Project";
 
-# Configured web paths & short URLs
-# This allows use of the /wiki/* path
-## https://www.mediawiki.org/wiki/Manual:Short_URL
-$wgScriptPath = "/w";        // this should already have been configured this way
-$wgArticlePath = "/wiki/$1";
-
-wfLoadSkin( 'Vector' );
-
 $wgEnableWikibaseRepo = false;
 $wgEnableWikibaseClient = true;
 
+# use jobs on request, client does not have a job runner
+$wgJobRunRate = 1;
```

Comment:
Good catch! How was this working before?

Reply:
Well, `1` is the default value for `wgJobRunRate`. This file here was overriding `LocalSettings.wbs.php` for the client instance before. So on client `wgJobRunRate` was never set.

Now, this file here does not become the `LocalSettings.wbs.php` in the client, but gets mounted to `LocalSettings.d/20_Wikibase.php`, overriding the original wikibase loading only. So the original `LocalSettings.wbs.php` is still in client, sets `wgJobRunRate` to `0`, then loads all the extensions from `LocalSettings.d` and therefore sets `wgJobRunRate` to `1` in the client case.